### ISSUE TITLE
"standard_init_linux.go:219: exec user process caused: exec format error' installing on Jetson

### ISSUE THREAD

**Issue by dadcoachengineer** [t0mer/deepstack-trainer#3]
jason@jetson01:~/deepstack_trainer$ sudo docker-compose up
Creating network "deepstack_trainer_default" with the default driver
Creating deepstack_trainer ... done
Attaching to deepstack_trainer
deepstack_trainer    | standard_init_linux.go:219: exec user process caused: exec format error
jason@jetson01:~/deepstack_trainer$ sudo docker ps
CONTAINER ID   IMAGE                           COMMAND                  CREATED          STATUS                        PORTS                  NAMES
ebf072f934a1   techblog/deepstack-trainer      "/usr/bin/python3 /o…"   12 seconds ago   Restarting (1) 1 second ago                          deepstack_trainer
6dfbffdb7611   deepquestai/deepstack:jetpack   "/app/server/server"     13 minutes ago   Up 13 minutes                 0.0.0.0:80->5000/tcp   pedantic_wiles

**Comment by t0mer** [comment]
Hi jasonshearer,
Try to pull the container with :arm tag.
the :latest does not support arm arch.

**Comment by dadcoachengineer** [comment]
Perfect. Just for reference here is my compose file running on a 2GB Jetson Nano

```
version: "3.7"
services:
  deepstack_engine:
    image: deepquestai/deepstack:jetpack-2021.06.1
    container_name: deepstack_engine
    privileged: true
    restart: always
    runtime: nvidia
    environment:
      - VISION-FACE=True
      - VISION-DETECTION=True
      - VISION-SCENE=True
      - NVIDIA_VISIBLE_DEVICES=all
    ports:
      - "80:5000"
    volumes:
      - deepstack_storage:/datastore


  deepstack_trainer:
    image: techblog/deepstack-trainer:arm
    container_name: deepstack_trainer
    privileged: true
    restart: always
    environment:
      - DEEPSTACK_HOST_ADDRESS=http://deepstack_engine:5000
      - MIN_CONFIDANCE=50
    ports:
      - "8080:8080"

volumes:
  deepstack_storage:
```

**Comment by t0mer** [comment]
Fixed in ver 1.0.0
Multi arch is now supported
